packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -192,21 +194,31 @@ const VisualizationProvider: FC<React.PropsWithChildren<Props>> = ({
         [calculateKeyColorAssignment, itemsMap],
     );
 
+    const isCalculateSeriesColorEnabled = useFeatureFlagEnabled(
+        FeatureFlags.CalculateSeriesColor,
+    );
+
     /**
      * Gets a shared color for a given series.
      */
     const getSeriesColor = useCallback(
         (seriesLike: SeriesLike) => {
-            if (seriesLike.color) return seriesLike.color;
+            if (seriesLike.color && !isGroupedSeries(seriesLike)) {
```

Comment:
I also added this check, because I noticed my chart had a series color (that was set when the chart was not grouped) . But this should not affect to the customer, as he didn't have this color set. Let me know if you want me to revert it.